### ISSUE TITLE
Op Generated code locally does not match master repository

### ISSUE THREAD

**Issue by JimClarke5** [tensorflow/java#381]
My local repository says it is 100% in sync with tensorflow/java master.
I do a `mvn clean`, `mvn install`,  then all the generated `Ops` are updated and next it wants me to recommit them.
Either something is wrong on my end, or the master repository is old.
I was expecting the locally generated classes to be the same as the main repository, as I am only working in `framework` at the moment.

@rnett Would this be something you might have a clue about.

Here is a sample of the delta between the repository and the locally generated classes.
`org.tensorflow.op.core.Abort`

**Stored on Master at `tensorflow/java`:**
```
  public static Abort create(Scope scope, Options... options) {
    OperationBuilder opBuilder = scope.env().opBuilder(OP_NAME, scope.makeOpName("Abort"));
    opBuilder = scope.apply(opBuilder);
   if (options != null) {
```

**Source generated locally:**
```
 public static Abort create(Scope scope, Options... options) {
    OperationBuilder opBuilder = scope.opBuilder(OP_NAME, "Abort");
    if (options != null) {
```

**Comment by rnett** [comment]
Yeah, that's what #375 is for.  Some of the ops weren't generated when they were added, and I forgot to commit the new generation for the init scope PR.

**Comment by JimClarke5** [comment]
Which is is supposed to be? The ones I generated locally or the ones on the repository?

**Comment by rnett** [comment]
Mostly what you generate locally, but apparently there's a few ops that are misclassified.
